experiments/compositions/samples/Eventarc/README.md

```diff
@@ -0,0 +1,61 @@
+# Eventarc with GCS Trigger - KCC Composition
+
+This repository provides a KCC Compositions approach to setting up an event-driven workflow that triggers a Cloud Workflow when objects are finalized in a Google Cloud Storage (GCS) bucket.
+
+**Key Components**
+
+* **KCC Composition:** Manages the creation of the Eventarc trigger and associated resources.
+* **Custom Resource Definition (CRD):** Defines a CRD for `EventarcConfig` to simplify Eventarc trigger configuration.
+* **Custom Resource (CR):** Provides a custom resource to specify the Eventarc trigger configuration.
+* **Storage Notification:** Configures a `StorageNotification` to publish events from the GCS bucket to a Pub/Sub topic.
+* **Service Account:** Creates a service account with necessary permissions for Eventarc and Pub/Sub.
+
+**Prerequisites**
+
+* **Google Cloud Project:**  With necessary APIs enabled (Cloud Storage, Pub/Sub, Eventarc, Cloud Workflows).
```

Comment:
Enabling APIs can be done by the composition as well.
PTAL: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/808d230c3464203f99eee1bd28a3054252f39a10/experiments/compositions/samples/CloudSQL/composition/hasql.yaml#L94